AirCasting/Utils/ShadowImages.swift

```diff
@@ -0,0 +1,28 @@
+// Created by Lunar on 15/10/2021.
+//
+
+import Foundation
+import UIKit
+import SwiftUI
+
+extension UIImage {
+    static let MainTabBarShadow = UIImage.gradientImageWithBounds(
+        bounds: CGRect(x: 0, y: 0, width: UIScreen.main.scale, height: 5),
+        colors: [
+            UIColor.clear.cgColor,
+            UIColor.black.withAlphaComponent(0.1).cgColor
+        ]
+    )
+    
+    static func gradientImageWithBounds(bounds: CGRect, colors: [CGColor]) -> UIImage {
+        let gradientLayer = CAGradientLayer()
+        gradientLayer.frame = bounds
+        gradientLayer.colors = colors
+        
+        UIGraphicsBeginImageContext(gradientLayer.bounds.size)
+        gradientLayer.render(in: UIGraphicsGetCurrentContext()!)
+        let image = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
+        return image!
```

Comment:
I'm not a fan of it, lots of force unwrapping, are we sure it won't crash? Especially as it's called in `onAppear` of our main view 😬

Reply:
Yeah, you are right. I am gonna to change it to if let, it was the fastest way during 'first stage' of code

Reply:
this is a pretty 50/50 scenario here. The reason it's optional is that those old functions like `UIGraphicsGetCurrentContext()` won't throw on error, they will return `nil` instead. But in this case `nil` value is actually an indication of error and here it doesn't look like we'll be having any errors ;) both `UIGraphicsGetCurrentContext` and `UIGraphicsGetImageFromCurrentImageContext` will probably be returning correct values all the time. IDK if propagating this optionality further is a good idea TBH